migrate_generator.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Migrate generator'
```

Comment:
Could you please add description so that it appears in UI ? 
`description: Module to generate migrations based on source CSV files.`

![image](https://user-images.githubusercontent.com/20738264/82904260-7d898f80-9f62-11ea-9e91-e396790c64b9.png)

Reply:
fixed